python/pip_install/repositories.bzl

```diff
@@ -21,8 +21,8 @@ _RULE_DEPS = [
     # START: maintained by 'bazel run //tools/private/update_deps:update_pip_deps'
     (
         "pypi__build",
-        "https://files.pythonhosted.org/packages/58/91/17b00d5fac63d3dca605f1b8269ba3c65e98059e1fd99d00283e42a454f0/build-0.10.0-py3-none-any.whl",
-        "af266720050a66c893a6096a2f410989eeac74ff9a68ba194b3f6473e8e26171",
+        "https://files.pythonhosted.org/packages/e2/03/f3c8ba0a6b6e30d7d18c40faab90807c9bb5e9a1e3b2fe2008af624a9c97/build-1.2.1-py3-none-any.whl",
```

Comment:
`pip_tools` v7.4.0 (coincidentlaly, the same https://github.com/jazzband/pip-tools/pull/1979 that I'm interested in using) also pinned `build >= 1.0.0`. In `pip_tools <= 7.3.0`, `build` did not have any version specifier.